### DISCUSSION TITLE
多语言文档能否实现根据客户端浏览器/系统语言来实现跳转自动语言

### DISCUSSION THREAD

**Discussion by Masterain98** [vuepress-theme-hope/vuepress-theme-hope#2919] [Q&A] [answered]
目前项目有中文（根目录`/`）和英文文档（`/en`），并且在有计划添加更多语言。请问能否实现根据特定的系统或浏览器语言来自动跳转到对应的目录？或者在用户第一次访问时，刻意地出现选择语言的窗口？

**Comment by Mister-Hope** [comment, marked as answer]
Could be a feature with redirect plugin.
